packages/openmrs/src/Adaptor.js

```diff
@@ -60,31 +60,33 @@ export function getPatient(uuid, callback = s => s) {
  * Make a get request to any OpenMRS REST endpoint.
  * @example
  * get("patient", {
- *   q: "Patient",
+ * query: {
+ *    q: "Patient",
  *   limit: 1,
+ * }
  * });
  * @function
  * @public
  * @param {string} path - Path to resource (excluding /ws/rest/v1/)
- * @param {object} query - parameters for the request
- * @param {function} [callback] - Optional callback to handle the response
+ * @param {object} options - parameters for the request
```

Comment:
Can we provide a link to the supported parameters here? If you look at our [commcare](https://docs.openfn.org/adaptors/packages/commcare-docs#bulk) docs, we link to the destination  docs to explain what's available.